.env.example

```diff
@@ -0,0 +1,12 @@
+DNANEXUS_TOKEN=
+SLACK_TOKEN=
+HTTP_PROXY=http://proxy.net.addenbrookes.nhs.uk:8080/
+HTTPS_PROXY=https://proxy.net.addenbrookes.nhs.uk:8080/
+ANSIBLE_GENETICDIR=/var/genetics
+ANSIBLE_LOGSDIR=/var/log/dx-streaming-upload
+ANSIBLE_SENDER=BioinformaticsTeamGeneticsLab@addenbrookes.nhs.uk
+ANSIBLE_RECEIVERS=abc@email.com,support@cuhbioinformatics.atlassian.net
+ANSIBLE_SERVER=smtp.net.addenbrookes.nhs.uk
```

Comment:
not sure we should be including addresses in github, can this env file just have the keys in and no values?

Reply:
updated